task-tracker/src/app/task.service.ts

```diff
@@ -1,30 +1,40 @@
-import { ReturnStatement } from "@angular/compiler";
 import { Injectable } from "@angular/core";
 
 @Injectable({
     providedIn: 'root'
 })
-
 export class TaskService {
     private tasks: any[] = [
-        { id: 1, dateCreate: '', priority: 'medium', description: 'Tidy up the room'},
-        { id: 2, dateCreate: '', priority: 'low', description: 'Sign up for a yoga class'},
-        { id: 3, dateCreate: '', priority: 'high', description: 'Read the article "10 Ways to Improve Remote Work Efficiency"'},
-        { id: 4, dateCreate: '', priority: 'medium', description: 'Do grocery shopping for the week'}
+        { id: 1, dateCreate: new Date().toISOString().split('T')[0], priority: 'medium', description: 'Tidy up the room', done: false },
+        { id: 2, dateCreate: new Date().toISOString().split('T')[0], priority: 'low', description: 'Sign up for a yoga class', done: false },
+        { id: 3, dateCreate: new Date().toISOString().split('T')[0], priority: 'high', description: 'Read the article "10 Ways to Improve Remote Work Efficiency"', done: false },
+        { id: 4, dateCreate: new Date().toISOString().split('T')[0], priority: 'medium', description: 'Do grocery shopping for the week', done: false }
     ];
-    constructor(){}
 
+    constructor() { }
 
     getTasks() {
         return this.tasks;
     }
 
     addTask(newTask: any) {
-        console.log(this.tasks)
-        newTask.id = this.tasks.length + 1; 
-        newTask.dateCreate = new Date();
+        newTask.id = this.tasks.length + 1;
```

Comment:
Jak utworzysz nowy interfejs (Task) to utwórz instancje obiektu przez 'object literal notation {}'.